husky_nodedrop:
  ip_addresses: ["192.168.1.11", "192.168.1.12", "192.168.1.13", "192.168.1.15", 127.0.0.1:5000]
  polling_rate: 1.0
  rssi_threshold: -25.0
